pkg/forwarder/internal/retry/HttpTransactionProto.proto

```diff
@@ -2,7 +2,7 @@
 
 syntax = "proto3";
 
-package forwarder;
+package retry;
```

Comment:
Why `internal/retry` and not just `retry` ?

Reply:
See https://golang.org/doc/go1.4#internalpackages.